### ISSUE TITLE
Display Deploy button and Console only if backend has connection to the cluster

### ISSUE THREAD

**Issue by mmelko** [KaotoIO/kaoto-ui#1118]
### Please describe the feature that you want to propose

Currently deploy and console are shown even if backend isn't connected which may be misleading.

**Comment by kahboom** [comment]
I think the reasoning was that we get back limited information from the cluster about why something isn't working, same for the custom resources. But maybe @Delawen can correct me here.

**Comment by mmelko** [comment]
@kahboom yeah, that's why I created https://github.com/KaotoIO/kaoto-backend/issues/410

**Comment by Delawen** [comment]
Currently the `/v1/capabilities?namespace=` endpoint in the backend provides information that can disable the deploy button depending on the dsl. There is a `deployable` attribute that is true only if the flow you are building can be deployed. Can't we use that?

We still have to improve that information coming from the backend (https://github.com/KaotoIO/kaoto-backend/issues/473 ) but I think that using that attribute in the frontend will fix this.

![image](https://user-images.githubusercontent.com/726590/220867701-ea802911-c91f-4d9e-afa0-b7e92bcd6fa8.png)

**Comment by kahboom** [comment]
@Delawen Yes that would be a good workaround.

**Comment by mmelko** [comment]
@Delawen so deployable is changing according the cluster connection isn't it? if yes then it can be used  but my impression is that it's used for DSL's that can be run in the cluster (everything except camel-route).

**Comment by Delawen** [comment]
**TL;DR: Please, use that to enable/disable the deploy button.**

Right now it only checks if it can or cannot be deployed via Camel K operator.

It will be soon (asap) extended to include if there is a cluster available, if that cluster supports deploying of that specific DSL,... 

And in the middle future it will include things like "oh, this is JBang deployable and you have JBang, go for it!"

So yes, please, use that attribute.

**Comment by kahboom** [comment]
@Delawen it seems this is already implemented: https://github.com/KaotoIO/kaoto-ui/blob/main/src/components/KaotoToolbar.tsx#L138
